mmeds/database/database.py

```diff
@@ -390,24 +390,65 @@ def get_col_values_from_table(self, column, table):
             data = cursor.fetchall()
         return data
 
-    def add_user(self, username, password, salt, email, privilege_level):
+    def add_user(self, username, password, salt, email, privilege_level, reset_needed=0):
         """ Add the user with the specified parameters. """
         with self.db.cursor() as cursor:
             cursor.execute('SELECT MAX(user_id) FROM user')
             user_id = int(cursor.fetchone()[0]) + 1
         # Create the SQL to add the user
-        sql = 'INSERT INTO `user` (user_id, username, password, salt, email, privilege)'
-        sql += ' VALUES (%(id)s, %(uname)s, %(pass)s, %(salt)s, %(email)s, %(privilege)s);'
+        sql = 'INSERT INTO `user` (user_id, username, password, salt, email, privilege, reset_needed)'
+        sql += ' VALUES (%(id)s, %(uname)s, %(pass)s, %(salt)s, %(email)s, %(privilege)s, %(reset_needed)s);'
 
         with self.db.cursor() as cursor:
             cursor.execute(sql, {'id': user_id,
                                  'uname': username,
                                  'pass': password,
                                  'salt': salt,
                                  'email': email,
-                                 'privilege': privilege_level
+                                 'privilege': privilege_level,
+                                 'reset_needed': reset_needed
                                  })
         self.db.commit()
+        Logger.info('USER {} HAS BEEN ADDED'.format(username))
+
+    def set_reset_needed(self, reset_needed):
+        """ Update the 'reset_needed' value, which forces the user to change their
+            password before continuing
+        """
+        if reset_needed is True:
+            reset_needed = 1
+        elif reset_needed is False:
+            reset_needed = 0
```

Comment:
```suggestion
        if reset_needed:
            reset_needed = 1
        else:
            reset_needed = 0
```

Reply:
Actually this is better

```suggestion
        reset_needed = int(reset_needed)
```